rj_gameplay/rj_gameplay/tactic/prep_move.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Tuple
+
+import numpy as np
+import stp
+from rj_msgs.msg import RobotIntent
+
+from rj_gameplay.role import dumb_move
+
+
+class PrepMove(stp.tactic.Tactic):
+    """Seeks to a single point, passed in on init."""
+
+    def __init__(self, world_state: stp.rc.WorldState):
+        super().__init__(world_state)
+
+        self._target_pt = np.array([0.0, 0.0])
+
+        self._role_requests.append(
+            (stp.role.cost.PickClosestToPoint(self._target_pt), dumb_move.DumbMove)
```

Comment:
again, you could pass in a specific ID here instead (but if the line is prioritized ahead of this one and that's given fixed robot_ids it wouldn't make a difference)